.ci/azure-pipelines.yml

```diff
@@ -21,6 +21,12 @@ steps:
   inputs:
     cmakeArgs: '.. -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DAMReX_SPACEDIM=3 -DAMReX_GPU_BACKEND=CUDA -DQUOKKA_PYTHON=OFF -G Ninja'
 
+- script: python ../src/StarCluster/perturbation.py --kmin=2 --kmax=64 --size=128 --alpha=2 --f_solenoidal=1.0
+  displayName: 'Run perturbation.py to generate input file for StarCluster test'
+
+- script: mv ../src/StarCluster/zdrv.hdf5 ../tests/
+  displayName: 'Move file to tests'
+
```

Comment:
Does this make sense @BenWibking

Reply:
I think you should put this in a Bash script inside `src/StarCluster/`.

Reply:
And then call the bash script here?

Reply:
No, call the Bash script from the add_test command, e.g. for SphericalCollapse:
https://github.com/quokka-astro/quokka/blob/685110f04c9f4dc9b1139a85e74c2cc9c0588e00/src/SphericalCollapse/CMakeLists.txt#L7

Just change the command.